cmd/kubelet/app/options/options.go

```diff
@@ -97,7 +97,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&s.FileCheckFrequency.Duration, "file-check-frequency", s.FileCheckFrequency.Duration, "Duration between checking config files for new data")
 	fs.DurationVar(&s.HTTPCheckFrequency.Duration, "http-check-frequency", s.HTTPCheckFrequency.Duration, "Duration between checking http for new data")
 	fs.StringVar(&s.ManifestURL, "manifest-url", s.ManifestURL, "URL for accessing the container manifest")
-	fs.StringVar(&s.ManifestURLHeader, "manifest-url-header", s.ManifestURLHeader, "HTTP header to use when accessing the manifest URL, with the key separated from the value with a ':', as in 'key:value'")
+	fs.StringVar(&s.ManifestURLHeader, "manifest-url-header", s.ManifestURLHeader, "HTTP header to use when accessing the manifest URL, with the key separated from the value with a ':', and the different header seperated with a ',' when more than one exist, as in 'key1:value1,kye2:value2'")
```

Comment:
This isn't backwards compatible because it changes the meaning of the existing flag (it would be interpreted differently).

You'll need to add a new StringArray flag (the one that accepts multiple values), and add a new field `ManifestURLHeaders`.  You'll have to combine this value with those.